.travis/deploy_spark_enrich.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
A little OT but is there a reason why we have all these individual `deploy_${foo}.sh` scripts, when we could just have a singular `deploy.sh ${foo}` approach?